libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java

```diff
@@ -146,8 +116,8 @@ public UserAccount(String authToken, String refreshToken,
 	public UserAccount(String authToken, String refreshToken,
```

Comment:
Should we put in a builder for 6.0? There are a lot of args in that constructor.

Reply:
Great idea! I'd prefer to do that in another PR later though.